Description: accept older revision of crate assert_cmd
 (upstream changes seem minor,
 so unreasonable to file a Debian bugreport about it)
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2025-08-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -123,7 +123,7 @@
 version-compare = { version = "0.1.1", optional = true }
 
 [dev-dependencies]
-assert_cmd = "2.0.13"
+assert_cmd = "2.0.12"
 cc = "1.0"
 chrono = "0.4.33"
 filetime = "0.2"
